tools/Make/board/imx8mp_iotgate.mk

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2023, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Set up variables for IMX8MQ_evk
+# Should be included _before_ toolchain makefile.
+PLATFORM := imx
+
+I2C_DRIV_DIR := ${PLATFORM}
+NET_DRIV_DIR := ${PLATFORM}
+ETH_DRIV := eth_driver_${PLATFORM}.elf
+ETH_DRIV_DIR1 := dwmac-5.10a
+ETH_DRIV_1 := eth_driver_dwmac-5.10a.elf
```

Comment:
This is not consistent, it's not clear what is what. There should be `ETH0` and `ETH1` probably.

Reply:
Most have only one network driver.  I don't really want to use '0' suffix when there's only one there.